portality/settings.py

```diff
@@ -111,12 +114,17 @@
         "editor_area",
         "assign_to_associate",
         "list_group_journals",
-        "list_group_suggestions"
+        "list_group_suggestions",
+        "api"
     ],
-    "associate_editor" : [
+    "associate_editor": [
         "edit_journal",
         "edit_suggestion",
-        "editor_area"
+        "editor_area",
+        "api"
+    ],
+    "user": [
+        #"api"              # added on account creation, commented out here so it can be revoked
```

Comment:
So that means that a publisher will always have the API role then? Lines 107-108 above